libs/ui/src/theme/HausThemeContext.tsx

```diff
@@ -20,22 +20,25 @@ type HausUI = {
   toggleLightDark: () => void;
 };
 
+type ProviderProps = {
+  children: ReactNode;
```

Comment:
I'm wondering if we want to outright allow for passing in multiple children in this since we're now adding the fragment. Something like:
`children: React.ReactNode|React.ReactNode[];`

@jordanlesich @alexkeating what ya'll think?

Reply:
I'm unsure how React handles this -- I'm wondering if the Fragment is treated as a single ReactNode child (and if so, then the Array is unnecessary in the type)